app/assets/javascripts/surveys.js

```diff
@@ -128,6 +128,7 @@
     init: function() {
       var activeSurvey = userSurveys.getActiveSurvey(userSurveys.defaultSurvey, userSurveys.smallSurveys);
       if (userSurveys.isSurveyToBeDisplayed(activeSurvey)) {
+        $('#global-bar').hide(); // Hide global bar if one is showing
```

Comment:
We currently have some fairly aggressive surveys running (ie every 2 pageviews). We should avoiding running these types of survey during a banner.